src/organisms/PunList/PunList.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Pun from "../../atoms/Pun";
+import "./PunList.css";
+import { PUNS } from "../../utils/enums";
+
+const PunList = () => {
+  localStorage.setItem("puns", JSON.stringify(PUNS));
+  const punsFromStorage = JSON.parse(localStorage.getItem("puns"));
```

Comment:
probably you're still going to do it (I didn't do it yet either), but the useLocalStorage hook is supposed to contain this logic